<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" hash="1451142286">
    <data-source source="LOCAL" name="Django default" uuid="9062d9fd-32b0-42b3-b120-bb8190c516ec">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/ubertool_eco/settings.py</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:C:\git\qed\ubertool_eco\db.sqlite3</jdbc-url>
      <explain-plan-config>
        <provider>com.intellij.database.plan.ExplainPlanProvider.NullExplainPlanProvider</provider>
      </explain-plan-config>
    </data-source>
    <data-source source="LOCAL" name="Django default2" uuid="fd1cb92b-f58c-4a31-98a6-d5b152dadc23">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/ubertool_eco/settings.py</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:C:\git\qed\ubertool_eco\db.sqlite3</jdbc-url>
      <explain-plan-config>
        <provider>com.intellij.database.plan.ExplainPlanProvider.NullExplainPlanProvider</provider>
      </explain-plan-config>
    </data-source>
  </component>
</project>